src/gluonts/ext/r_forecast/_univariate_predictor.py

```diff
@@ -165,11 +178,14 @@ def _override_params(
             params["output_types"] = ["mean"]
         elif self.method_name in UNIVARIATE_QUANTILE_FORECAST_METHODS:
             params["output_types"] = ["quantiles", "mean"]
+            params_intervals = self.params['intervals'] if "intervals" in self.params else list()
             if intervals is None:
-                # This corresponds to quantiles: 0.05 to 0.95 in steps of 0.05.
-                params["intervals"] = list(range(0, 100, 10))
+                # This corresponds to quantiles: 0.05 to 0.95 in steps of 0.05, and other quantiles provided as inputs
+                intervals = list(range(0, 100, 10)) + params_intervals
             else:
-                params["intervals"] = np.sort(intervals).tolist()
+                intervals = intervals + params_intervals
+            intervals = list(dict.fromkeys(intervals))
+            params["intervals"] = np.sort(intervals).tolist() 
```

Comment:
@leica2023 I don't understand this change, could you help clarify? The original code appears simple: it uses a set of given intervals if provided, and otherwise falls back to a default choice. Doesn't this give all the flexibility needed here alread?